tests/unit/modules/test_slackware_service.py

```diff
@@ -0,0 +1,234 @@
+"""
```

Comment:
we are still in the process of migrating entirely to pytest, but since this is a new test file i would love to see it using pytest instead in `tests/pytests/unit/modules` so we dont have to migrate it later if you're willing to do so.

Reply:
Oh, ok. I will change it, as it's a new feature it's better to born in the right way.

Reply:
thanks!

Reply:
Ok, I guess it's done.